modules/tensor_mechanics/src/materials/ComputeReturnMappingStress.C

```diff
@@ -38,8 +37,8 @@ ComputeReturnMappingStress::computeQpStress()
   // compute the stress increment (as a tensor) and the inelastic strain increment (tensor) required to return to the yield surface
```

Comment:
Would you mind updating this comment to clarify that the whole stress, not the stress increment, is returned please?